CONTRIBUTING.md

```diff
@@ -11,6 +11,7 @@
         -   [Create a Minimal Working Example](#create-a-minimal-working-example)
     -   [Ask for Community Support](#ask-for-community-support)
 -   [Contribute Code](#contribute-code)
+-   [Contribute to Documentation](#contribute-to-documentation)
 
 ## The TestCafe Community
 
```

Comment:
Since I'm unable to edit line 31 below, can you please update it for me?

Line 31:
In the last sentence of this paragraph, I don't think "the presence of" is necessary. Can we update this line as follows?

`TestCafe is a [Node.js](https://nodejs.org/en/) application. It supports [all actively maintained](https://github.com/nodejs/Release#release-schedule) versions of the Node.js framework. The installation process requires the [node package manager](https://www.npmjs.com/) (npm) software utility.`